example/ionic-angular-v6/ios/App/App/capacitor.config.json

```diff
@@ -9,6 +9,8 @@
 	"webDir": "www",
 	"packageClassList": [
 		"SentryCapacitor",
-		"SentryCapacitor"
+		"kSentryLevelNone",
+		"kSentryTransactionNameSourceCustom",
+		"SentryRRWebEvent"
```

Comment:
q: why are those classes added?

Reply:
That is a good question, it seems to be only added on later versions of capacitor automatically.
I will investigate later what's up with it but so far it doesn't affect the sdk usage.